### ISSUE TITLE
Security: Set 'security' passwords from standard input instead of CLI arg

### ISSUE THREAD

**Issue by Ambrevar** [atlas-engineer/nyxt#731]
In `password-security`:

```
(defmethod save-password ((password-interface security-interface) &key password-name password service)
  (uiop:run-program (list *security-cli-program* "add-internet-password"
                          "-a" password-name "-s" service "-w" password)))
```

`password` should be set from standard input (see other password interfaces) because command line arguments are visible to all users on a system.  In other words, this leaks the password.

**Comment by Ambrevar** [comment]
Not necessarily "standard input", it can be something else as long as it does not leak the password to other users on the system.

**Comment by jmercouris** [comment]
Hm, I can take a look. Good point!

**Comment by Ambrevar** [comment]
@jmercouris Any change you could fix this before the prerelease?
Otherwise I suggest we disable this password interface for the time being.

**Comment by jmercouris** [comment]
There is a chance, yes :-D

**Comment by Ambrevar** [comment]
`security` does not seem to be able to read from standard input, so for now let's disable it.
Done with 2e6c5d7d52c7d6c24616af9de216cb3551d845cf.

**Comment by aadcg** [comment]
Is the description still up to date? Seems critical.

**Comment by jmercouris** [comment]
It is not actually critical, because good luck running Nyxt on macOS today :-D

**Comment by jmercouris** [comment]
Yes, this is a program that only runs on macOS, used for interacting with the keychain

**Comment by aadcg** [comment]
Stale and irrelevant at the moment.
